packages/netstring/README.md

```diff
@@ -0,0 +1,12 @@
+# netstring
+
+This is an implementation of asynchronous streams framed as netstrings.
```

Comment:
acknowledge 1997 spec?

Netstrings
D. J. Bernstein, djb@pobox.com
19970201
https://cr.yp.to/proto/netstrings.txt